sdk/identity/identity/test/public/node/caeARM.spec.ts

```diff
@@ -15,10 +15,10 @@ import {
   createEmptyPipeline,
   createPipelineRequest,
 } from "@azure/core-rest-pipeline";
-import { DeveloperSignOnClientId } from "../../../src/constants.js";
-import { IdentityClient } from "../../../src/client/identityClient.js";
+import { DeveloperSignOnClientId } from "../../../dist/esm/constants.js";
+import { IdentityClient } from "../../../dist/esm/client/identityClient.js";
```

Comment:
This is a public test and I wonder why are these imported from internal code.

Reply:
I wasn't aware that we're not able to - so that's a little surprising to me. With that said, identity has been around fro a while and drifts in a few ways from the rest of the packages I suppose. With that said, just like everywhere else we should be importing from `src` not `dist`